server/manifest.go

```diff
@@ -7,5 +7,5 @@ var manifest = struct {
 	Version string
 }{
 	ID:      "com.mattermost.welcomebot",
-	Version: "1.2.0",
```

Comment:
Let's leave this out for now. The version will be changed in its own PR after numerous PRs have been merged.